app/build.gradle

```diff
@@ -310,19 +292,18 @@ dependencies {
     implementation "org.mozilla.components:search:$mozilla_components_version"
     implementation "org.mozilla.photon:colors:$mozilla_components_version"
 
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:0.22.5"
 
     focusImplementation 'com.adjust.sdk:adjust-android:4.11.4'
     focusImplementation 'com.google.android.gms:play-services-analytics:10.2.6' // Required by Adjust
 
-    // What I actually want to define here is geckoArmCompile and geckoX86Compile. But gradle doesn't
-    // allow me to define dependencies for such a flavor combination. However as we are not building
-    // WebView + (ARM / x86) flavor combinations this should be good enough for now.
     armImplementation "org.mozilla:geckoview-beta-armeabi-v7a:${geckoview_version}"
     x86Implementation "org.mozilla:geckoview-beta-x86:${geckoview_version}"
     aarch64Implementation "org.mozilla:geckoview-beta-arm64-v8a:${geckoview_version}"
 
+    implementation 'com.jakewharton:process-phoenix:2.0.0'
```

Comment:
This patch doesn't seem to use this library yet?

Reply:
Yes, I split the work into two PRs. The part that restarts the app isn't required yet.